bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -157,6 +157,12 @@
 				<type>openweathermap:moon-phase</type>
 			</add-channel>
 		</instruction-set>
+		<instruction-set targetVersion="3">
```

Comment:
You also need to set the thingtypeversion to the thing declaration.

Reply:
Done